x-pack/plugins/aiops/public/components/log_rate_analysis/log_rate_analysis_results.tsx

```diff
@@ -118,8 +120,8 @@ const columnSearchAriaLabel = i18n.translate('xpack.aiops.analysis.columnSelecto
 const columnsButton = i18n.translate('xpack.aiops.logRateAnalysis.page.columnsFilterButtonLabel', {
   defaultMessage: 'Columns',
 });
-const fieldsButton = i18n.translate('xpack.aiops.analysis.fieldFilterButtonLabel', {
-  defaultMessage: 'Filter fields',
+const fieldsButton = i18n.translate('xpack.aiops.analysis.fieldsButtonLabel', {
+  defaultMessage: 'Fields',
```

Comment:
I'm unsure if this label is clear enough? Maybe `Analyze fields`? But that could imply that the button will perform some sort of action to analyze fields in the data, and `Fields to analyze` is probably too long!

But there is the info text inside the popover so maybe the concise label is ok. Any thoughts?

Reply:
Yes, had some similar thoughts and then ended up with the most concise `Fields` given that we have the explanation in the popover. Maybe `Analysis fields`? Internally in the code we call them `Field candidates` and that phrase is also used in the loading status, maybe use that, wdyt?

Reply:
I'm not convinced that `Field candidates` is any clearer. With the text in the popover, I think best stick with the concise `Fields`.